vsintegration/src/FSharp.Editor/CodeFixes/AddInstanceMemberParameter.fs

```diff
@@ -11,7 +11,7 @@ open Microsoft.CodeAnalysis.CodeFixes
 open CancellableTasks
 
 [<ExportCodeFixProvider(FSharpConstants.FSharpLanguageName, Name = CodeFix.AddInstanceMemberParameter); Shared>]
-type internal FSharpAddInstanceMemberParameterCodeFixProvider() =
+type internal AddInstanceMemberParameterCodeFixProvider() =
```

Comment:
Mef might be confused if there's another type with the same name is loaded in codefixes (despite being in different namespace).
Not sure if it still applies.

Reply:
Any way to check that? Running things locally still works.

Reply:
way to check that:

Create two codefixes in different namespace, but sharing their name.

Reply:
Yeah, I tried this out, looks like this works. Like, I just copypasted the code of a code fix, put it into a different namespace, adjusted the names and both show up. 

![image](https://github.com/dotnet/fsharp/assets/5451366/f155c742-813f-407a-a5d6-e21289c559fd)

So it doesn't matter how the types are called. That said, strings matter, if both fixes have the same "title", both will be triggered but only one will show up. This is unrelated to the changes in the PR, just good to know.